Watermark On Image/Watermark_on_image.py

```diff
@@ -0,0 +1,25 @@
+#Import required Image library
+from PIL import Image, ImageDraw, ImageFont
+
+#Create an Image Object from an Image
+im = Image.open('test_image.jpg')
+width, height = im.size
+
+draw = ImageDraw.Draw(im)
+text = "Amazing Python Scripts"
+
```

Comment:
@aishwaryachand  take the text input from the user

Reply:
changes made ,pls check